rtrlib/rtr_mgr.h

```diff
@@ -269,8 +271,8 @@ enum aspa_status rtr_mgr_verify_as_path(struct rtr_mgr_config *config, uint32_t
  * @return SPKI_SUCCESS On success
  * @return SPKI_ERROR If an error occurred
  */
-int rtr_mgr_get_spki(struct rtr_mgr_config *config, const uint32_t asn, uint8_t *ski, struct spki_record **result,
-		     unsigned int *result_count);
+enum rtr_spki_rtvals rtr_mgr_get_spki(struct rtr_mgr_config *config, const uint32_t asn, uint8_t *ski,
```

Comment:
maybe prefix this kind of functions, which subsystem they will access.

so something like: `rtr_mgr_bgpsec_get_spki`